packages/framework-integration-tests/integration/aws/utils.ts

```diff
@@ -258,3 +263,37 @@ export async function countSnapshotItems(): Promise<number> {
 export async function sleep(ms: number): Promise<void> {
   return new Promise((resolve) => setTimeout(resolve, ms))
 }
+
+//
+export async function waitForIt<TResult>(
+  tryFunction: () => Promise<TResult>,
+  checkResult: (result: TResult) => boolean,
+  tryEveryMs = 1000,
```

Comment:
I'd call this `millisInterval`, from this name I infer it is a `bool`

Reply:
The idea was to have a name like: "Try every X ms"